.github/workflows/pr-agent.yml

```diff
@@ -1,20 +0,0 @@
-on:
```

Comment:
These code changes removed a workflow that ran on pull requests that were opened, reopened, or ready for review, as well as issue comments. The job named "pr_agent_job" was removed, which ran on every pull request event as long as the sender was not a bot. It used the Codium-ai/pr-agent action with environment variables for OPENAI_KEY and GITHUB_TOKEN. The permissions for issues, pull requests, and contents were also specified. 

Overall, these changes remove the automation that was triggered by pull requests and issue comments, specifically the PR Agent action that responded to user comments on pull requests.